js/ui/html_editor/ui.html_editor.js

```diff
@@ -117,7 +117,7 @@ const HtmlEditor = Editor.inherit({
             */
             /**
             * @name dxHtmlEditorToolbar.items
-            * @type Array<dxHtmlEditorToolbarItem,string>
+            * @type Array<dxHtmlEditorToolbarItem,Enums.HtmlEditorToolbarItem>
```

Comment:
`Array<dxHtmlEditorToolbarItem,string,Enums.HtmlEditorToolbarItem>`?

Reply:
We need `string` only if custom strings are accepted. AFAIK, it isn't so.